cmd/verrazzano-operator/main.go

```diff
@@ -38,61 +38,70 @@ var (
 	startController              bool
 )
 
+const apiVersionPrefix = "/20210501"
+
 // homePage provides a default handler for unmatched patterns
 func homePage(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "verrazzano-operator apiserver")
 	fmt.Println("GET /")
 }
 
 // handleRequests registers the routes and handlers
-func handleRequests(apiServerFinished chan bool) {
-	myRouter := mux.NewRouter().StrictSlash(true)
+func handleRequests(apiServerFinished chan bool, rootRouter *mux.Router) {
+	apiRouter := registerPathHandlers(rootRouter)
+
+	glog.Info("Starting API server...")
+
+	if apiServerRealm == "" {
+		glog.Warning("Bearer token verification is disabled as apiServerRealm is not specified")
+	}
+
+	glog.Fatal(http.ListenAndServe(":3456", apiserver.CORSHandler(apiserver.AuthHandler(apiRouter))))
```

Comment:
Does this use the prefix?  If not, should it?

Reply:
The prefix is automatically used by the fact of us using the apiRouter as the request router